examples/syscalls/Makefile

```diff
@@ -10,4 +10,4 @@ clean:
 $(BINS): Makefile
 
 %: %.c
-	gcc $(CFLAGS) $< -o $@
+	gcc $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) $< -o $@
```

Comment:
These are C programs, so do we really want `$CPPFLAGS`?

Reply:
Since the C compiler uses the C preprocessor, I think that it makes sense to also include C preprocessor flags.

Reply:
Oops, I'm sorry. I thought `CPPFLAGS` was for C++.